app/src/main/res/values/strings.xml

```diff
@@ -15,6 +15,7 @@
     <string name="discard">Discard</string>
     <string name="save">Save</string>
     <string name="issuer">Issuer</string>
+    <string name="yandex_pin">Pin (4–16 digits)</string>
```

Comment:
We should probably capitalize **PIN** any time we display it in the UI.

Reply:
Changed :)